include/pmix_common.h.in

```diff
@@ -1471,6 +1471,9 @@ static inline void* pmix_calloc(size_t n, size_t m)
         (a)->rank = (c);                        \
     }while(0)
 
+#define PMIX_XFER_PROCID(a, b)      \
```

Comment:
To be consistent with other `XFER` options, shouldn't this be `PMIX_PROCID_XFER`?

Reply:
I chose to be consistent across the procid macros - it is `PMIX_LOAD_PROCID`, `PMIX_CHECK_PROCID`, etc., so...